pkg/services/sqlstore/stats.go

```diff
@@ -89,74 +90,80 @@ func GetSystemStats(query *models.GetSystemStatsQuery) error {
 
 	query.Result = &stats
 
-	return err
+	return nil
 }
 
-func roleCounterSQL(role, alias string) string {
-	return `
+func roleCounterSQL(role string, alias string, onlyActive bool) string {
+	var sqlQuery = `
 		(
 			SELECT COUNT(DISTINCT u.id)
-			FROM ` + dialect.Quote("user") + ` as u, org_user
-			WHERE ( org_user.user_id=u.id AND org_user.role='` + role + `' )
-		) as ` + alias + `,
+			FROM ` + dialect.Quote("user") + ` AS u, org_user
+			WHERE u.last_seen_at > ? AND ( org_user.user_id=u.id AND org_user.role='` + role + `' )
```

Comment:
While you're working with this code, maybe we should use the `dialect.Quote` function used in the line above for this line as well (+ same in the other query below)?

```suggestion
			WHERE u.last_seen_at > ? AND ( org_user.user_id=u.id AND org_user.role=` + dialect.Quote(role) + ` )
```

Reply:
dialect.Quote seems for table name, not for string, I don't think this would work. It may be used on other queries on this file though, I update that.